feature_engine/selection/drop_high_PSI_features.py

```diff
@@ -0,0 +1,460 @@
+
+        variables: list
+            The list of variables to evaluate. If None, the transformer will
+            evaluate all numerical variables in the dataset.
+
+        switch_basis: boolean
+            If set to true the role of the two matrices involved in the PSI
+            calculations (basis and measurement) will be switch. This is an
+            important option as the PSI value is not symmetric
+            (i.e. PSI(a, b) != PSI(b, a)).
+
+        threshold: float
+            Threshold above which the distribution of a feature has changed so
+            much that the feature will be dropped. The most common values are
+            0.25 (large shift) and 0.10 (medium shift).
+
+        method: string or callable
+            Type of binning used to represent the distribution of the feature.
+            In can be either "equal_width" for equally spaced bins or
+            "equal_frequency" for bins based on quantiles.
+
+        n_nbins: int
+            Number of bins used in the binning. For numerical feature a value of
+            10 is considered as appropriate. For features with lower cardinality
+            lower values are usually used.
+
+        min_pct_empty_buckets: float
+            Value to add to empty bucket (when considering percentages). If a bin
+            is empty the PSI value may jump to infinity. By adding a small number
+            to empty bins, this issue is avoided. If the value added is too large,
+            it may disturb the calculations.
+
+        missing_values: str
+            Takes values 'raise' and 'ignore'. Whether the missing values should
+            be raised as error or ignored when determining correlation.
+
+        Returns:
+            None
+
+        """
+        if not isinstance(n_bins, int) or n_bins <= 1:
+            raise ValueError("n_bins must be an integer larger than 1.")
+
+        if not isinstance(switch_basis, bool):
+            raise ValueError("The value of switch basis must be True or False.")
+
+        if not isinstance(threshold, (float, int)) or threshold < 0:
+            raise ValueError("threshold must be a float larger than 0")
+
+        if not isinstance(basis, (pd.DataFrame, dict)):
+            raise ValueError("basis must be either a pd.DataFrame or a dict")
+
+        if isinstance(basis, dict):
+            if "date_col" not in basis.keys() or "cut_off_date" not in basis.keys():
+                raise ValueError("date_col and cut_off_date must be keys")
+
+        if (
+            not isinstance(min_pct_empty_buckets, (float, int))
+            or min_pct_empty_buckets < 0
+        ):
+            raise ValueError("min_pct_empty_buckets must be larger or equal to 0")
+
+        if missing_values not in ["raise", "ignore", "include"]:
+            raise ValueError(
+                "missing_values can only be 'raise', 'ignore' or 'include'."
+            )
+        if method.lower() in ["equalwidth", "equal_width", "equal width"]:
```

Comment:
I think parameters should accept only one value, we should not try and interpret what the user tried to do. It is very nice of you, but I think in the long run it may cause problems, for example backwards compatibility. 

What values are allowed, are explained in the docstings and we could help with an error message here

Reply:
Solved by keeping only the original names equal_width and equal_frequency